x/auth/ante/sigverify.go

```diff
@@ -196,10 +196,6 @@ func OnlyLegacyAminoSigners(sigData signing.SignatureData) bool {
 }
 
 func (svd SigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
-	// no need to verify signatures on recheck tx
```

Comment:
This is not correct. We can and should bypass signature verification on rechecking.